Feature/alaska prod enable es

dougszumski opened the pull request.

dougszumski committed:
Enable ElasticSearch

dougszumski committed:
Add un-customised overcloud components

dougszumski committed:
Move Kibana and ElasticSearch to monitoring group

This creates the following diff between the overcloud-components
before they were customised and after this change was applied.

@@ -94,21 +94,21 @@

 [karbor:children]
 control

 [kibana:children]
-control
+monitoring

 [telegraf:children]
 compute
 control
 monitoring
 network
 storage

 [elasticsearch:children]
-control
+monitoring

 [haproxy:children]
 network

 [hyperv]

markgoddard commented on etc/kayobe/kolla/inventory/overcloud-components.j2:
```diff
@@ -0,0 +1,201 @@
+# This inventory section provides a mapping of the components to top level
```
Perhaps you could outline why we're using a custom inventory here, or alternatively add a comment next to lines that are changed from the original.

dougszumski replied:
Will do. I put a diff in the commit message to check nothing unexpected happened.

markgoddard replied:
Oh, yes I see now. I should make more effort to look at the individual commits, especially when you've gone to the effort of separating them nicely.

dougszumski replied:
No worries - github seems good at hiding the commit messages.

dougszumski committed:
Make it clear which mappings were customised

The components Kibana and Elasticsearch are used as part of the
monitoring infrastructure. A separate monitoring server is used
to host these.
